docs/usage/INVENTORY_SYNC.md

```diff
@@ -4,122 +4,169 @@ Utility which provides API for building CTP inventory update actions and invento
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-  - [Usage](#usage)
-    - [Build all update actions](#build-all-update-actions)
-    - [Build particular update action(s)](#build-particular-update-actions)
-    - [Sync list of inventory entry drafts](#sync-list-of-inventory-entry-drafts)
-  - [Under the hood](#under-the-hood)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Usage
+- [Usage](#usage)
+  - [Sync list of inventory entry drafts](#sync-list-of-inventory-entry-drafts)
+    - [Prerequisites](#prerequisites)
+    - [Running the sync](#running-the-sync)
+  - [Build all update actions](#build-all-update-actions)
+  - [Build particular update action(s)](#build-particular-update-actions)
+- [Caveats](#caveats)
 
-### Build all update actions
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-<!-- TODO: Probably #14 affects inventory sync as well. Ensure before providing the code snippet. -->
+## Usage
 
-### Build particular update action(s)
+### Sync list of inventory entry drafts
 
-To build the update action for changing inventory quantity:
+<!-- TODO - GITHUB ISSUE#138: Split into explanation of how to "sync from project to project" vs "import from feed"-->
 
+#### Prerequisites
+1. The sync expects a list of `InventoryEntryDraft` objects that have their `sku` fields set,
+   otherwise the sync will trigger an `errorCallback` function set by the user (more on it can be found down below in the options explanations).
+
+2. Every inventory entry may have a reference to a supply `Channel` and a reference to the `Type` of its custom fields. These
+   references are matched by their `key`. Therefore, in order for the sync to resolve the actual ids of those references,
+   their `key`s has to be supplied in the following way:
+   - Provide the `key` value on the `id` field of the reference. This means that calling `getId()` on the
+   reference would return its `key`. 
```

Comment:
I guess this will change if we accept resourceIdentifier now?

Reply:
Yes, this would change with the new refactoring of reference resolution.